samples/sample13/README.md

```diff
@@ -11,143 +11,240 @@
+        │   │   │       └── cool_drink.yaml 
+        |   ├── drink_store
+        |   |   ├── drink_store.yaml
+        │   |   └── drink-store-deployment
+        │   │   │   ├── Chart.yaml
+        │   │   │   └── templates                              
+        │   │   │       └── drink_store.yaml 
+        |   └── hot_drink
+        |   |   ├── hot_drink.yaml
+        │   |   └── hot-drink-deployment
+        │   │   │   ├── Chart.yaml
+        │   │   │   └── templates                              
+        │   │   │       └── hot_drink.yaml 
+        ├── balo                                               
+            ├── cool_drink-2019r3-java8-0.0.1.balo
+            ├── drink_store-2019r3-java8-0.0.1.balo                   
+            └── hot_drink-2019r3-java8-0.0.1.balo                    
+        ├── bin                                                
+            ├── cool_drink.jar     
+            ├── drink_store.jar                                       
+            └── hot_drink.jar                                      
+        └── caches                                             
+            ├── bir_cache                                      
+            │   ├── kathy                                       
+            │   │   ├── cool_drink                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── cool_drink.bir
+            │   │   ├── drink_store                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── drink_store.bir                     
+            │   │   └── hot_drink                                  
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── hot_drink.bir                      
+            └── jar_cache                                      
+                └── kathy                                       
+            │   │   ├── cool_drink                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-cool_drink-0.0.1.jar
+            │   │   ├── drink_store                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-drink_store-0.0.1.jar                     
+            │   │   └── hot_drink                                  
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-hot_drink-0.0.1.jar
   
     ```
 ### How to run:
-1. Go to `resource/docker` folder and run the build.sh file. This will generate two databases deployment in mysql namespace.
+1. Use the link [https://jar-download.com/artifacts/mysql/mysql-connector-java/8.0.11/source-code] to download [mysql-connector-java-8.0.11.jar] and add it to a new folder named `libs` inside sample13 directory.
```

Comment:
This is not how we put a hyperlink in markdown right ?

Reply:
changed accordingly.